test/components/preprocessors/test_document_splitter.py

```diff
@@ -494,3 +498,301 @@ def test_run_document_only_whitespaces(self):
         doc = Document(content="  ")
         results = splitter.run([doc])
         assert results["documents"][0].content == "  "
+
+
+class TestSplittingNLTKSentenceSplitter:
```

Comment:
Just to confirm did all tests from the test_nltk_document_splitter.py file make it into here?

Reply:
not, only the ones that over NLTK-specific code since there were also tests, in NLTKDocumentSplitter, to cover the same functionality as in DocumentSplitter

Reply:
I might have missed something, I will double check